composer.json

```diff
@@ -106,14 +110,19 @@
         "Fix multilingual site's layout edit context issue": "https://www.drupal.org/files/issues/2019-12-19/3101231-19.patch",
         "SQLite database locking errors cause fatal errors": "https://www.drupal.org/files/issues/1120020-59.patch",
         "Error: Call to a member function getLabel() after enabling layout_builder": "https://www.drupal.org/files/issues/2019-11-14/2985882-entityfield-43.patch",
-        "Split Interface Translation permission": "https://www.drupal.org/files/issues/2020-01-14/locale-administer-locale-permission-3067056-7.patch"
+        "Split Interface Translation permission": "https://www.drupal.org/files/issues/2020-01-14/locale-administer-locale-permission-3067056-7.patch",
+        "Layout builder doesn't show new \"extra fields\"": "https://www.drupal.org/files/issues/2020-04-16/missing_extra_fields-3069578-12.patch",
+        "POST /quickedit/metadata returning HTTP 503": "https://www.drupal.org/files/issues/2020-01-09/2893407-39.patch"
       },
-      "drush/drush": {
-        "Import multiple custom translation po files": "https://patch-diff.githubusercontent.com/raw/drush-ops/drush/pull/4251.patch"
+      "drush/page_manager": {
```

Comment:
IMO it is better to keep patches ordered by package name, the same as require section so better move this down the list.